packages/@expo/cli/e2e/__tests__/login-test.ts

```diff
@@ -46,6 +46,7 @@ it('runs `npx expo login --help`', async () => {
         -u, --username <string>  Username
         -p, --password <string>  Password
         --otp <string>           One-time password from your 2FA device
+        -s, --sso                Log in with SSO
```

Comment:
Seems this was disabled

Reply:
It was removed from the help to hide it as SSO has been released, but we want it available or dogfooding